src/components/QuizQuestion/QuizQuestion.tsx

```diff
@@ -0,0 +1,98 @@
+import React, { useState } from "react"
+import {
+    QuizQuestionAndAnswers
+} from "../../types/index"
+import "./quiz-question.css"
+
+const QuizQuestion = (props: QuizQuestionAndAnswers) => {
+    const {
+        questionsDetails: {
+            question,
+            options
+        },
+        questionNumber,
+        setSelectedOptionResult,
+        isQuizSubmitted
+    } = props
+    const [isOptionSelected, setIsOptionSelected] = useState([false, false, false])
+
+    const selectQuestionOption = (event:React.MouseEvent, option:{optionValue: string,isCorrectOption: boolean}, optionIndex:number) => {
+        setSelectedOptionResult((prevResponseArray:Boolean[]):Boolean[]=>{
```

Comment:
suggestion: use `boolean` here.

Read this for more info: https://www.tutorialsteacher.com/typescript/typescript-boolean

Reply:
TypeScript was giving me error when I tried boolean, hence I did this 😄